packages/@aws-cdk/aws-iotevents/lib/detector-model.ts

```diff
@@ -27,6 +43,38 @@ export interface DetectorModelProps {
    */
   readonly detectorModelName?: string;
 
+  /**
+   * A brief description of the detector model.
+   *
+   * @default none
+   */
+  readonly detectorModelDescription?: string;
+
+  /**
+   * Information about the order in which events are evaluated and how actions are executed.
+   *
+   * When setting to SERIAL, variables are updated and event conditions are evaluated in the order
+   * that the events are defined.
+   * When setting to BATCH, variables within a state are updated and events within a state are
+   * performed only after all event conditions are evaluated.
+   *
+   * @default EvaluationMethod.BATCH
+   */
+  readonly evaluationMethod?: EvaluationMethod;
+
+  /**
+   * The value used to identify a detector instance. When a device or system sends input, a new
+   * detector instance with a unique key value is created. AWS IoT Events can continue to route
+   * input to its corresponding detector instance based on this identifying information.
+   *
+   * This parameter uses a JSON-path expression to select the attribute-value pair in the message
+   * payload that is used for identification. To route the message to the correct detector instance,
+   * the device must send a message payload that contains the same attribute-value.
+   *
+   * @default - none (single detector instance will be created and all inputs will be routed to it)
+   */
+  readonly key?: string;
```

Comment:
This name, however, is way, way too generic. How about calling it `detectorKey`, or something like that?

Reply:
`detectorKey` is good! Hmm... I should think for more understandability instead of CFn copy. Thank you!